hiss/application/tests/admin_tests/application.py

```diff
@@ -47,3 +47,13 @@ def test_creates_accurate_rsvp_deadline(self):
         deadline = timezone.now().replace(hour=23, minute=59, second=59, microsecond=0)
         create_rsvp_deadline(self.user, deadline)
         self.assertEquals(self.user.rsvp_deadline, deadline)
+
+    def test_export_emails(self):
```

Comment:
You'll probably want to update this test name to correspond with the new function name.